Library/Homebrew/test/Gemfile

```diff
@@ -1,9 +1,15 @@
 source "https://rubygems.org"
 
-gem "mocha", "~> 1.1"
-gem "minitest", "~> 5.3"
-gem "rake", "~> 10.3"
-gem "parallel_tests", "~> 2.9"
+group :test do
```

Comment:
Given these are only used for tests this seems a bit redundant but don't have strong feelings.